.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,35 +1,11 @@
-
-### Description of changes: 
-
-Describe Kani's current behavior and how your code changes that behavior. If there are no issues this PR is resolving, explain why this change is necessary.
-
-### Resolved issues:
+> Please ensure your PR description includes the following:
+> 1. Description on how your changes improve Kani.
+> 2. Provide some context on the problem you are solving.
+> 3. List issues that are resolved by this PR.
```

Comment:
```suggestion
> 3. A list of issues that are resolved by this PR.
```

Reply:
Do we even need this item? We already have `Resolves #ISSUE-NUMBER` below.